lib/auth/permissions.go

```diff
@@ -338,6 +338,7 @@ func GetCheckerForBuiltinRole(clusterName string, recConfig types.SessionRecordi
 						types.NewRule(types.KindSessionRecordingConfig, services.RO()),
 						types.NewRule(types.KindClusterAuthPreference, services.RO()),
 						types.NewRule(types.KindSemaphore, services.RW()),
+						types.NewRule(types.KindNetworkRestrictions, services.RW()),
```

Comment:
Why does the node need `RW` permissions to this resource? If my reading is correct, it's a singleton resource created using `tctl` commands and nodes only need to read from it. So `services.RO()` should be sufficent?